### DISCUSSION TITLE
Live Wallpaper Choose Weather Type

### DISCUSSION THREAD

**Discussion by trymeouteh** [breezy-weather/breezy-weather#1458] [Q&A] [answered]
### Describe your idea

To have a setting in the live wallpaper to choose the weather it will display all the time regardless of the actual weather the app is pulling from the APIs.

This way one can always have it sunny, rainy, snowing, overcast, etc.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment, marked as answer]
Tap the settings icon on top right of Breezy Weather live wallpaper settings
